content/pl/disposability.md

```diff
@@ -0,0 +1,14 @@
+## IX. Disposability
+### Zwiększ elastyczność aplikacji przez szybki start i bezproblemowe zamknięcie
+
+**[Procesy](./processes) aplikacji 12factor są *jednorazowe*, znaczy to, że mogą być wystartowane lub zatrzymane** Ułatwia to elastyczne skalowanie i szybkie wdrożenia [kodu](./codebase), zmianę [konfiguracji](./config) i zapewnia większą stabilność przy deploymencie na produkcję.
+
+Procesy powinny dążyć do **minimalizowania czasu swojego rozruchu**.  W idealnej sytuacji, proces potrzebuje kilka sekund by wystartować i być gotowy na przyjmowanie zapytań oraz zadań. Krótki czas startu zapewnia większą szybkość dla skalowania i [wypuszczania](./build-release-run) procesów. Zwiększa to również zdolności aplikacji do radzenia sobie z problemami, ponieważ manager procesów może łatwo przenieść je na nową maszynę fizyczną będzie taka potrzeba.
```

Comment:
1. `W idealnej sytuacji proces powinien potrzebować kilku sekund na to aby wystartować i być gotowym na przyjmowanie zapytań`
2. `Większą szybkość dla skalowania i wypuszczania procesów`. Nie brzmi to po polsku.
3. `manager procesów` -> polsko angielsko tutaj też nie pasuje. 
4. `jeśli bedzię taka potrzeba` -> `jeśli zajdzie taka potrzeba`